features/schedule/basic_workflow/feature.go

```diff
@@ -0,0 +1,122 @@
+package basic_workflow
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk-features/harness/go/harness"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/workflow"
+)
+
+var Feature = harness.Feature{
+	Workflows: BasicScheduleWorkflow,
+	Execute:   Execute,
+}
+
+func BasicScheduleWorkflow(ctx workflow.Context, arg string) (string, error) {
+	return arg, nil
+}
+
+func Execute(ctx context.Context, r *harness.Runner) (client.WorkflowRun, error) {
+	// Start schedule every 2s
+	workflowID := uuid.NewString()
+	handle, err := r.Client.ScheduleClient().Create(ctx, client.ScheduleOptions{
+		// TODO(cretz): Required? https://github.com/temporalio/sdk-go/issues/957
+		ID:      uuid.NewString(),
```

Comment:
nit: I'd suggest following the test harness' naming convention for schedule id (`${this.source.relDir}-${randomUUID()}`), and `${scheduleId}-workflow` for the Workflow ID.

Reply:
> following the test harness' naming convention for schedule id 

The harness doesn't have a naming convention for schedule ID. I don't think it matters much the shape of the identifier in these cases so long as unique. Notice the workflow type is the function name not prefixed with its qualified path to help the UI either.